client/src/components/common/Card.svelte

```diff
@@ -33,6 +40,7 @@
     <span class="circle rating {extended && 'extend'}">{rating}/5</span>
     <span class="circle time {extended && 'extend'}">{time} min</span>
   </button>
+  <button class="details" on:click={showRecipeRedirect}>{recipeId}</button>
```

Comment:
card doesn't need to know the recipe id. It can dispatch an event called "details" and the parent component of the card calls the showRecipeRedirect

Reply:
look at how the delete and unpublish have been done already